Sources/PointFree/Tasks.swift

```diff
@@ -0,0 +1,249 @@
+    SimpleEmailLayoutData(
+      user: user,
+      newsletter: .welcomeEmails,
+      title: subject,
+      preheader: "",
+      template: .default,
+      data: user
+    )
+  }
+}
+
+private let wrapper = { view in
+  [
+    emailTable([style(contentTableStyles)], [
+      tr([
+        td([valign(.top)], [
+          div([`class`([Class.padding([.mobile: [.all: 0], .desktop: [.all: 2]])])], view)
+          ])
+        ])
+      ])
+  ]
+}
+
+func welcomeEmail1(_ user: Database.User) -> Email {
+  let subject = "Thanks for signing up to Point-Free!"
+  return prepareWelcomeEmail(
+    to: user,
+    subject: subject,
+    content: welcomeEmailView(subject, welcomeEmail1Content)
+  )
+}
+
+let welcomeEmail1Content = View<Database.User> { user -> [Node] in
+  [
+    markdownBlock(
+      """
+      👋 Howdy!
+
+      It's been a week since you signed up for [Point-Free](\(url(to: .home))). We hope you've learned
+      something new about functional programming, and maybe even introduced it into your codebase!
+
+      We'd love to [have you as a subscriber](\(url(to: .pricing(nil, expand: nil)))), so please let us know if
+      you have any questions. Just reply to this email!
+      """
+    ),
+    user.episodeCreditCount > 0
+      ? markdownBlock(
+        """
+        In the meantime, it looks like you have a **free episode credit**! You can use this to see *any*
+        subscriber-only episode, completely for free! Just visit [our site](\(url(to: .home))), go to an
+        episode, and click the "\(useCreditCTA)" button!
+
+        Users typically spend their credit on these popular episodes:
+
+        * [Dependency Injection Made Easy](https://www.pointfree.co/episodes/ep16-dependency-injection-made-easy)
+        * [Algebraic Data Type: Part 1](https://www.pointfree.co/episodes/ep4-algebraic-data-types)
+        * [Tagged](https://www.pointfree.co/episodes/ep12-tagged)
+        * [Contravariance](https://www.pointfree.co/episodes/ep14-contravariance)
+        """
+        )
+      : nil
+    ,
+    markdownBlock(
+      """
+      When you're ready to subscribe for yourself _or_ your team, visit
+      [our subscribe page](\(url(to: .pricing(nil, expand: nil))))!
+      """
+    ),
+    subscribeButton,
+    ]
+    .compactMap(id)
+}
+
+func welcomeEmail2(_ user: Database.User) -> Email {
+  let subject = "Free episodes on Point-Free"
+  return prepareWelcomeEmail(
+    to: user,
+    subject: subject,
+    content: welcomeEmailView(subject, welcomeEmail2Content)
+  )
+}
+
+let welcomeEmail2Content = View<Database.User> { user -> [Node] in
+  let freeEpisodeLinks = Current.episodes()
+    .filter { !$0.subscriberOnly }
+    .map {
+      """
+      * [\($0.title)](\(url(to: .episode(.left($0.slug)))))
+      """
+    }
+    .joined(separator: "\n")
+  return [
+    markdownBlock(
+      """
+      👋 Hey there!
+
+      You signed up for a Point-Free account a couple weeks ago but still haven't subscribed!
```

Comment:
how about link `Point-Free` here since the homepage is not linked anywhere else in the body?